lib/utils/report.mjs

```diff
@@ -49,10 +51,10 @@ export default function report(problem) {
 	}
 
 	const { ignoreDisables } = result.stylelint.config || {};
+	const { disabledRanges } = result.stylelint;
```

Comment:
```suggestion
	const { disabledRanges, config: { ignoreDisables = {} } } = result.stylelint;
```

Reply:
Unfortunately, the suggested code emits the type error below...

```
TS2339: Property 'ignoreDisables' does not exist on type 'Config | undefined'
```

Reply:
I think you read my comment before my edit.